resources/prosody-plugins/mod_measure_message_count.lua

```diff
@@ -64,6 +64,10 @@ function send_event(room)
         polls_count = room._muc_polls_count or 0;
     };
 
+    if room.created_timestamp then
+        event_properties.duration = (os.time() * 1000 - room.created_timestamp) / 1000;
```

Comment:
Do we need millisecond precision for the event? I'd suggest rounding to the nearest second or even minute

Reply:
I think this is in seconds

Reply:
I misread 🤦